allocate.go

```diff
@@ -401,6 +421,14 @@ func CombinedOutput(w io.Writer) ExecAllocatorOption {
 	}
 }
 
+// WSURLReadTimeout is used to set a timeout for reading websocket url on
+// chrome process starting.
+func WSURLReadTimeout(timeout time.Duration) ExecAllocatorOption {
```

Comment:
I don't think we want to make this configurable for now. A constant of 10s should be plenty for any machine to start Chrome, yet still be short enough that we don't hang a process for a long time.

Make the 10s a constant instead of a field, too.